src/theme/theme.ts

```diff
@@ -37,6 +37,17 @@ const generatePalette = (
 export const palette: PaletteOptions = {
   ...generatePalette(ui, 'ui'),
   ...generatePalette(collage, 'collage'),
+  mode: 'light',
+  primary: themeColors.primary,
+  secondary: themeColors.secondary,
+  error: themeColors.error,
+  warning: themeColors.warning,
+  success: themeColors.success,
```

Comment:
Instead of all these you can simply destructure the entire object like this
```suggestion
...{ themeColors },
```